addons/payment_authorize/models/authorize_request.py

```diff
@@ -138,9 +138,9 @@ def create_customer_profile(self, partner, transaction_id):
 
         payment = response.get('paymentProfile', {}).get('payment', {})
         if self.payment_method_type == 'credit_card':
-            res['name'] = payment.get('creditCard', {}).get('cardNumber')
+            res['payment_details'] = payment.get('creditCard', {}).get('cardNumber')[-4:]
```

Comment:
Why not store what you received since the field is meant for "the clear part of the payment details"?

Reply:
Because they return `cardNumber` and `accountNumber` in this format: `XXXX1111`